example/docker-compose/etc/tempo-azure.yaml

```diff
@@ -0,0 +1,52 @@
+auth_enabled: false
+
+server:
+  http_listen_port: 3100
+
+distributor:
+  receivers:                           # this configuration will listen on all ports and protocols that tempo is capable of.
+    jaeger:                            # the receives all come from the OpenTelemetry collector.  more configuration information can
+      protocols:                       # be found there: https://github.com/open-telemetry/opentelemetry-collector/tree/master/receiver
+        thrift_http:                   #
+        grpc:                          # for a production deployment you should only enable the receivers you need!
+        thrift_binary:
+        thrift_compact:
+    zipkin:
+    otlp:
+      protocols:
+        http:
+        grpc:
+    opencensus:
+
+ingester:
+  trace_idle_period: 10s               # the length of time after a trace has not received spans to consider it complete and flush it
+  traces_per_block: 100                # cut the head block when it his this number of traces or ...
+  max_block_duration: 5m               #   this much time passes
+
+compactor:
+  compaction:
+    compaction_window: 1h              # blocks in this time window will be compacted together
+    max_compaction_objects: 1000000    # maximum size of compacted blocks
+    block_retention: 1h
+    compacted_block_retention: 10m
+    flush_size_bytes: 5242880
+
+storage:
+  trace:
+    backend: azure                        # backend configuration to use
+    wal:
+      path: /tmp/tempo/wal             # where to store the the wal locally
+      bloom_filter_false_positive: .05 # bloom filter false positive rate.  lower values create larger filters but fewer false positives
+      index_downsample: 10             # number of traces per index record
+    azure:
+      container_name: tempo                    # how to store data in azure
+      endpoint_suffix: azurite
+      storage_account_name: "devstoreaccount1"
+      storage_account_key: "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
```

Comment:
How are we obtaining these? Is there a way to start the `azurite` container with a default account/pass like in the minio example?

Reply:
These are the default name/key values for azurite per the [docs](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator#authorize-with-shared-key-credentials), so it does not need to be defined on the azurite container first.

Reply:
Yes, those are the defaults, but we can still override them using "AZURITE_ACCOUNTS="account1:key1""